Neos.Neos/Classes/Fusion/Helper/BackendHelper.php

```diff
@@ -31,6 +31,7 @@ class BackendHelper implements ProtectedContextAwareInterface
      *
      *     Translation.id("mh").locale(Neos.Backend.interfaceLanguage()).translate()
      *
+     * @return string
```

Comment:
Why?

Reply:
for the doc generation in eel otherwise it would be missing the return